packages/universal-components/src/RangeDatePicker/DayItemArrow/DayItemArrow.js

```diff
@@ -51,20 +49,32 @@ export default class DayItemArrow extends React.Component<Props> {
     });
   }
 
+  renderArrows = () => (
+    <View style={styles.touchableContainer}>
+      {isAnimationEnabled && (
+        <Animated.View
```

Comment:
Animation is desynchronised after selection. When arrow is clicked both animations should be restarted.

![arr-sync](https://user-images.githubusercontent.com/2660330/55624741-1aa9c780-57a7-11e9-8d29-91de0f999605.gif)

Reply:
Yes, I also noticed this issue. Animation value should be shared from `RenderDay` component and then it should be fine. But I plan to fix it in next PR, not to let this PR make bigger.

Reply:
Definitely, it's quite minor issue for now. Thanks